docker-compose.yml

```diff
@@ -8,11 +8,18 @@ services:
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=postgres
   web:
-    build: .
-    command: python3 manage.py migrate --noinput && python3 manage.py runserver 0.0.0.0:8000
+    build: ./src/django
+    # We would normally have something like ./scripts/console that would allow running
+    # migrations manually, and then only have the runserver in here.
```

Comment:
We also usually run migrations when the user executes `./scripts/setup` and/or `./scripts/update`